app/Filament/Resources/SearchRecipeResource.php

```
@@ -23,21 +24,110 @@
+                // Filters can be added here, like for category or price range
             ])
             ->actions([
                 Tables\Actions\EditAction::make(),
+
+                Action::make('launch_search')
+                    ->label('Launch Search')
+                    ->icon('heroicon-o-play')
+                    ->color('primary')
+                    ->requiresConfirmation()
+                    ->action(function (SearchRecipe $record): void {
+                        LaunchSearchRecipeLaravelJob::dispatch($record);
```

PHPStan (2.0.3)

Job class App\Jobs\LaunchSearchRecipeJob constructor invoked with 1 parameter in App\Jobs\LaunchSearchRecipeJob::dispatch(), 0 required.


(arguments.count)